### PR TITLE
bounty: enable submit only once per user

### PR DESCRIPTION
**Pull request by thelostone-mc**
##### Description

- Checks if logged in user has already fulfilled bounty & hides submit button
- remove dead code

##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)
- bounty actions 

##### Testing
x

##### Refers/Fixes

- https://github.com/gitcoinco/web/issues/2118

### CODE REVIEW

File: app/assets/v2/js/pages/bounty_details.js [hunk 1]
```diff
@@ -777,20 +775,19 @@ var do_actions = function(result) {
...
   let show_github_link = result['github_url'].substring(0, 4) == 'http';
-  let show_submit_work = is_open;
+  let show_submit_work = is_open && !has_fulfilled;
   let show_kill_bounty = !is_status_done && !is_status_expired && !is_status_cancelled && isBountyOwner(result);
   let show_job_description = result['attached_job_description'] && result['attached_job_description'].startsWith('http');
   const show_increase_bounty = !is_status_done && !is_status_expired && !is_status_cancelled;
   const submit_work_enabled = !isBountyOwner(result);
-  const start_stop_work_enabled = !isBountyOwner(result);
-  const increase_bounty_enabled = isBountyOwner(result);
-  let show_accept_submission = isBountyOwner(result) && !is_status_expired && !is_status_done;
```
Comment (nemaniarjun) [inline app/assets/v2/js/pages/bounty_details.js]: Umm, why was this removed?

Comment (nemaniarjun) [reply app/assets/v2/js/pages/bounty_details.js]: Just saw the PR description. Ignore this comment.

Comment (nemaniarjun) [reply app/assets/v2/js/pages/bounty_details.js]: We should add something like https://github.com/jendrikseipp/vulture to the linting stage for the django code.

File: app/assets/v2/js/pages/bounty_details.js [hunk 2]
```diff
@@ -777,20 +775,19 @@ var do_actions = function(result) {
   // Find interest information
   const is_interested = is_current_user_interested(result);
 
+  const has_fulfilled = result['fulfillments'].filter(fulfiller => fulfiller.fulfiller_github_username === document.contxt['github_handle']).length > 0;
+
```
Comment (octavioamu) [inline app/assets/v2/js/pages/bounty_details.js]: not directly related to this task, but whats happens in the app if the user changes his username in github?  it also changes in gitcoin to the urls (profile/username)  and username in every place?

Comment (thelostone-mc) [reply app/assets/v2/js/pages/bounty_details.js]: Oooo that would mess up everything at our end 🤔 
cc @mbeacom 

we would have to build that out to update all references 🤔
